android-components/components/compose/cfr/src/main/java/mozilla/components/compose/cfr/CFRPopupFullscreenLayout.kt

```diff
@@ -510,8 +510,22 @@ internal class CFRPopupFullscreenLayout(
             format = PixelFormat.TRANSLUCENT
             flags = WindowManager.LayoutParams.FLAG_LAYOUT_IN_SCREEN or
                 WindowManager.LayoutParams.FLAG_HARDWARE_ACCELERATED
+            title = "${getApplicationName()} ${anchor.resources.getString(R.string.mozac_cfr_a11y_notification_title)}"
         }
 
+    /**
+     * Get the name of the application to use it as accessibility title for the CFR Popup.
+     */
+    private fun getApplicationName(): String {
+        val applicationInfo = anchor.context.applicationInfo
+        val stringId = applicationInfo.labelRes
+        return if (stringId == 0) {
+            applicationInfo.nonLocalizedLabel.toString()
+        } else {
+            anchor.context.getString(stringId)
+        }
+    }
```

Comment:
so turns out we apparently already have an extension function to help us with this

[`val Context.appName: String`](https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/support/ktx/src/main/java/mozilla/components/support/ktx/android/content/Context.kt#56-57)